### ISSUE TITLE
How to add modules to 

### ISSUE THREAD

**Issue by Thorazin** [TheZoq2/dcs_on_linux#29]
Hello everyone, I have DCS open Beta running apparently just fine on Ubuntu 24.04 it starts and I can fly around in the TF-51 and SU-25 just fine. I am unable to purchase any modules and add them however. I press the buy button in the module manager and nothing happens.

Here is what Lutris says about my system:

Vulkan support: YES
Esync support: YES
Fsync support: YES
Wine installed: YES
Gamescope: NO
Mangohud: NO
Gamemode: YES
Steam: YES
In Flatpak: NO
[System]
OS: Ubuntu 24.04 Noble Numbat
Arch: x86_64
Kernel: 6.8.0-31-generic
Desktop: ubuntu:GNOME
Display Server: wayland
[CPU]
Vendor: AuthenticAMD
Model: AMD Ryzen 9 7950X 16-Core Processor
Physical cores: 16
Logical cores: 32
[Memory]
RAM: 62.5 GB
Swap: 8.0 GB
[Graphics]
Vendor: AMD
OpenGL Renderer: AMD Radeon RX 7900 XTX (radeonsi, navi31, LLVM 17.0.6, DRM 3.57, 6.8.0-31-generic)
OpenGL Version: 4.6 (Compatibility Profile) Mesa 24.0.5-1ubuntu1
OpenGL Core: 4.6 (Core Profile) Mesa 24.0.5-1ubuntu1
OpenGL ES: OpenGL ES 3.2 Mesa 24.0.5-1ubuntu1
Vulkan Version: 1.3.275
Vulkan Drivers: AMD Radeon RX 7900 XTX (RADV NAVI31) (1.3.274)

I am using wine version wine-ge-8-26 and on the stable channel.

Any other info you need I will provide.

**Comment by TheZoq2** [comment]
Are you trying to buy modules from inside the game? I think the way that has worked for me is to buy them on the ED website and then add them using the in-game module manager

**Comment by Thorazin** [comment]
Yes I was trying to buy in game. I do not see a mechanism to add modules any other way.

**Comment by Thorazin** [comment]
D'oh, I got it. Downloading the Tomcat now.
